src/components/tabs/tabs.ts

```diff
@@ -448,8 +448,8 @@ export class Tabs extends Ion implements AfterViewInit {
     // walk backwards through the tab selection history
     // and find the first previous tab that is enabled and shown
     console.debug('run previousTab', this._selectHistory);
-    for (var i = this._selectHistory.length - 2; i >= 0; i--) {
```

Comment:
Noted.

I'm actually going to close this PR since the problem is actually a bit more complex than I initially thought. See here for reference: https://github.com/driftyco/ionic/issues/9999#issuecomment-272311148

Or would it be better to keep this one since it's already open and make all the future changes here?